docs/source/AdministratorGuide/HowTo/index.rst

```diff
@@ -15,3 +15,4 @@ FIXME: These sections describes things
    multiVO
    pitExport
    dedicateddfc
+   supercomputers
```

Comment:
Maybe this should go to the `AdministratorGuide/Resources` instead of HowTos?

Reply:
Why do you think that it should go to the `AdministratorGuide/Resources` section?

`AdministratorGuide/Resources` seems to contain explanations about `src/DIRAC/Resources`, while `supercomputers.rst` is more about how we use DIRAC components to interact with supercomputers (how to).
But if you think it is not at the right place, I will move it.

Reply:
The resources section is about external resources for various purposes

from the resources index:

> DIRAC can incorporate resources provided by existing Grid infrastructures (e.g. WLCG, OSG) as well as sites not integrated in any grid infrastructure [...]

So this kind of fits the description for the super computer section.

Reply:
Oh I see, I didn't read the `index` sorry.
I moved `supercomputers.rst` in `AdministratorGuid/Resources`.